chef_master/source/resource_remote_file.rst

```diff
@@ -371,11 +375,26 @@ This resource has the following properties:
 
    Enable If-Modified-Since headers. Set to ``false`` to disable If-Modified-Since headers. To use this setting, ``use_conditional_get`` must also be set to ``true``. Default value: ``true``.
 
-``show_progess`` 
+``show_progess``
    **Ruby Types:** TrueClass, FalseClass
 
    Displays the progress of the file download. Set to ``true`` to enable this feature. Default value: ``false``.
 
+``remote_user``
+   **Ruby Types:** String
+
+   *Windows only:* The user name of a user with access to the remote file specified by the `source` property. Default value: `nil`. The user name may optionally be specifed with a domain, i.e. `domain\user` or `user@my.dns.domain.com` via Universal Principal Name (UPN) format. It can also be specified without a domain simply as `user` if the domain is instead specified using the `remote_domain` attribute. Note that this property is ignored if `source` is not a UNC path. If this property is specified, the `remote_password` property **must** be specified.
```

Comment:
"The name of a user with access to the remote file"

"The domain may also be set using the `remote_domain` property."

"If this property is specified, the `remote_password` property is **required**."